akka-docs/rst/java/stream/stream-io.rst

```diff
@@ -100,7 +100,7 @@ Akka Streams provide simple Sources and Sinks that can work with :class:`ByteStr
 on files.
 
 .. note::
-  Since the current version of Akka (``2.3.x``) needs to support JDK6, the currently provided File IO implementations
+  Since the version of Akka (``2.3.x``) needs to support JDK6, the currently provided File IO implementations
   are not able to utilise Asynchronous File IO operations, as these were introduced in JDK7 (and newer).
   Once Akka is free to require JDK8 (from ``2.4.x``) these implementations will be updated to make use of the
```

Comment:
The entire note needs updating, as we are now on 2.4.x and JDK8. We just haven't had time to move to NIO. Maybe we should just remove it.